client/luigi-client.js

```diff
@@ -169,8 +169,8 @@
       return id;
     },
     /**
-     * Removes a init listener
-     * @param {string} id the id that was returned by `addInitListener`
+     * Removes an init listener.
+     * @param {string} id the id that was returned by the `addInitListener` function
```

Comment:
`id` or `ID`? There is an inconsistency compared to the line 200

Reply:
let's stay with `id`.